samples/net/openthread/coap_server/README.rst

```diff
@@ -52,3 +78,5 @@ In addition, it uses the following Zephyr libraries:
 * :ref:`zephyr:kernel`:
 
   * ``include/kernel.h``
+
+OpenThread CoAP API documentation is available `here <https://openthread.io/reference/group/api-coap>`_.
```

Comment:
Isn't this needed for the client sample as well? If so, make it into a project-wide link in `links.txt`.
See "Hyperlinks" in https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/doc_styleguide.html for details.

Reply:
No, As stated in the last line of the overview both samples demonstrate different api usage. This one demonstrates api provided by openthread while the client sample provides native zephyr api usage.
For those two samples we have decided to demonstrate both possibilities. In the future examples we plan to stick only with Zephyrs CoAP API, however it is still poorly implemented and needs some rework.